### DISCUSSION TITLE
Bug in CODEOWNERS file viewer

### DISCUSSION THREAD

**Discussion by rajbos** [community/community#14251] [Other Feature Feedback, Questions, & Ideas]
There is a bug in validating the CODEOWNERS file after which it gives of an expectation that is not valid.

The viewer says that this file is valid:
![image](https://user-images.githubusercontent.com/6085745/161550583-68c97629-8ab2-4765-ac47-71af8f25a22a.png)

Where it should be having a path annotation or a '*' in front of the line, so that it actually does something:

``` yaml
*  @Microsoft-Bootcamp/attendees_0404
```

The one in the screenshot will not assign any PR's with a CODEOWNERS check in the branch policy to that team.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by rajbos** [reply]
Done. Feels like a black box though, will I get notified if this is (being) addressed?

**Reply by MathiasReker** [reply]
Yes, the bug-report will be taken into account.

**Reply by rajbos** [reply]
I got feedback from the bug report that this is indeed by design, although not super clear for me at the beginning


> You're correct -- that's actually its practical application! In an owned folder, you can exclude a single file within the folder by leaving its owners empty.
>
> In this example, @octocat owns any file in the /apps directory in the root of your repository except for the /apps/github subdirectory, as its owners are left empty.
>
> /apps/ @octocat
> /apps/github
>
>We've included this example here:
>
> https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file

I'll see if I can help make the docs a bit more clear on this :-D.

**Reply by ccayg-sainsburys** [reply]
That's not great feedback - the documented example is:
```
pattern owner
pattern
```
The bug (as you reported) is that
```
owner
```
is considered valid
